WebApplication1/UserTaskList.aspx

```diff
@@ -127,16 +127,17 @@
                                 <asp:ListItem>Normal</asp:ListItem>
                                 <asp:ListItem>High</asp:ListItem></asp:DropDownList>
                           </div><br/>
-						  <p><br/>Current Progress:</p><div class="col-xs-6"><asp:TextBox Type="number" min="0" required ID="curProg" runat="server" class="form-control" />
+						  <p><br/>Current Progress:</p><div class="col-xs-6"><asp:TextBox Type="number" min="0" value ="0" required ID="curProg" runat="server" class="form-control" />
                           </div><br/>
-						  <p><br/>End Progress:</p><div class="col-xs-6"><asp:TextBox Type="number" min="1" required onchange="validate()" ID="endProg" runat="server" class="form-control" />
+						  <p><br/>End Progress:</p><div class="col-xs-6"><asp:TextBox Type="number" min="1" value="1" required ID="endProg" runat="server" class="form-control" />
                           </div>
+                              <asp:CompareValidator runat="server" id="cmpNumbers" controltovalidate="curProg" controltocompare="endProg" operator="LessThan" type="Integer" errormessage="End Progress has to be higher than Current Progress" />
```

Comment:
CompareValidator is a easy way to do the comparison in-line for the aspx instead of getting messy with C# code. Nice!